modules/workflow-workflowoperation/src/main/java/org/opencastproject/workflow/handler/workflow/TagWorkflowOperationHandler.java

```diff
@@ -127,8 +141,21 @@ public WorkflowOperationResult start(WorkflowInstance workflowInstance, JobConte
         element.setIdentifier(null);
         element.setURI(e.getURI()); // use the same URI as the original
       }
-      if (configuredTargetFlavor != null)
-        element.setFlavor(MediaPackageElementFlavor.parseFlavor(configuredTargetFlavor));
+
+      if (configuredTargetFlavor != null) {
+        String targetFlavor = configuredTargetFlavor;
+
+        if (wildcardInTargetFlavorType) {
+          targetFlavor = targetFlavor.replaceFirst("\\*", element.getFlavor().getType());
```

Comment:
This call escapes the *, the following one does not.  They should be the same, right?